pkg/pmem-csi-driver/main.go

```diff
@@ -51,6 +57,24 @@ func Main() int {
 
 	klog.V(3).Info("Version: ", version)
 
+	var client *kubernetes.Clientset
+	if *schedulerListenAddr != "" {
+		if DriverMode(*mode) != Controller {
+			pmemcommon.ExitError("scheduler listening", errors.New("only supported in the controller"))
+			return 1
+		}
+		config, err := rest.InClusterConfig()
```

Comment:
Can you please get this Kubernetes specific code out of this package.

Reply:
> How to get connected to Kubernetes is specific to the command. For example, the command might have specific command line options for specifying the configuration and which cluster to use.

Holding the configuration option is ok in this package, but I am concerned about the creating the clientset using that configuration.

Reply:
Does 998d940f address this?